src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -1,29 +1,69 @@
 import * as Location from 'expo-location';
-import React from 'react';
+import React, { useCallback } from 'react';
+import { UseFormSetValue } from 'react-hook-form';
 import { StyleSheet, View } from 'react-native';
 
 import { device, normalize, texts } from '../../../config';
 import { useLocationSettings, useSystemPermission } from '../../../hooks';
 import { MapMarker } from '../../../types';
 import { LoadingSpinner } from '../../LoadingSpinner';
+import { RegularText } from '../../Text';
 import { Wrapper } from '../../Wrapper';
 import { Input } from '../../form';
 import { Map } from '../../map';
 import { getLocationMarker } from '../../settings';
-import { RegularText } from '../../Text';
 
 export const SueReportLocation = ({
   control,
   selectedPosition,
-  setSelectedPosition
+  setSelectedPosition,
+  setValue
 }: {
   control: any;
   selectedPosition: Location.LocationObjectCoords | undefined;
   setSelectedPosition: (position: Location.LocationObjectCoords | undefined) => void;
+  setValue: UseFormSetValue<{
+    city: string;
+    description: string;
+    email: string;
+    firstName: string;
+    houseNumber: string;
+    images: string;
+    lastName: string;
+    phone: string;
+    street: string;
+    termsOfService: boolean;
+    title: string;
+    zipCode: string;
+  }>;
```

Comment:
Do we need all these variables here? Can't we only add what we use in this component?

Reply:
No, if i reduce them:

> Argument of type 'UseFormSetValue<{ city: string; description: string; email: string; firstName: string; houseNumber: string; images: string; lastName: string; phone: string; street: string; termsOfService: boolean; title: string; zipCode: string; }>' is not assignable to parameter of type 'UseFormSetValue<{ city: string; houseNumber: string; street: string; zipCode: string; }>'.

Reply:
Okay, then it stays. 👍